boards/nxp/frdm_mcxc444/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BOARD_FRDM_MCXC444
+
+config WDT_DISABLE_AT_BOOT
```

Comment:
I'm not sure this is the right way to set this config. Looking at the code base, there is HAS_WDT_DISABLE_AT_BOOT config that is usually set in the drivers/watchdog/Kconfig.mcux. 

For reference, see: https://github.com/zephyrproject-rtos/zephyr/blob/9aecdd47c77fab1b21235ae2bc564a3d0dcf9769/drivers/watchdog/Kconfig.stm32#L12